modules/virt-building-vm-containerdisk-image.adoc

```diff
@@ -131,7 +131,7 @@ EOF
 +
 [source,terminal]
 ----
-$ virt-customize -a <UUID>.qcow2 --run=customize-vm --selinux-relabel
+$ virt-customize -a <UUID>-disk.qcow2 --run=customize-vm --selinux-relabel
```

Comment:
nit: In the Dockerfile bellow the `uuid` is in lower case.
Please consider aligning them.